linuxdoc fails to produce dvi or ps from the sample SGML, but has no 
problem with the SGML sources in xorg-server.  Build PDF instead (which 
is more useful anyway) until this is figured out.

--- origsrc/linuxdoc-tools-0.9.66/doc/Makedoc.sh	2009-11-09 13:58:25.000000000 -0600
+++ src/linuxdoc-tools-0.9.66/doc/Makedoc.sh	2010-03-24 19:39:43.863141100 -0500
@@ -49,10 +49,10 @@ fi
 if [ -n "`which latex`" ]; then
 	ln -s $TMPDIR/linuxdoc $TMPDIR/sgml2latex
 	echo "- Building latex docs" >&2
-	$TMPDIR/sgml2latex --pass="\usepackage{times}" -o dvi ./guide
+	$TMPDIR/sgml2latex --pass="\usepackage{times}" -o pdf ./guide
 fi
 
-if [ -n "`which dvips`" ]; then
+if false; then
 	echo "   + dvips" >&2
 	dvips -t letter -o ./guide.ps ./guide.dvi
 	if [ -n "`which gzip`" -a -f ./guide.ps ]; then
